### ISSUE TITLE
What is the learning rate？

### ISSUE THREAD

**Issue by ShuchunXu** [silvercherry/Improving-Robust-Fairness-via-Balance-Adversarial-Training#2]
Dear Authers:
I found that the training loss value is 'nan' after using the default training parameters in the source code, what are the training parameters used by the author, and the focus is on what is the learning rate?
![屏幕截图 2024-09-17 163428](https://github.com/user-attachments/assets/6a66bed6-216a-4912-ad84-51b867e422c1)

**Comment by ShuchunXu** [comment]
![屏幕截图 2024-09-17 163428](https://github.com/user-attachments/assets/2153da2e-a89e-433f-8cb3-3e302eb89ec1)

**Comment by ShuchunXu** [comment]
How do I download the supplementary materials?
